src/main/java/com/tvtracker/enterprise/dto/MediaEntry.java

```diff
@@ -1,11 +1,9 @@
 package com.tvtracker.enterprise.dto;
 
-import lombok.Getter;
-import lombok.Setter;
+import lombok.Data;
 
-@Getter
-@Setter
-public class MediaEntry {
+public @Data
+class MediaEntry {
```

Comment:
The @Data annotation bundles @ToString, @EqualsAndHashCode, @Getter on all fields, @Setter on all non-final fields, and @RequiredArgsConstructor. 
https://projectlombok.org/features/Data

Reply:
Good.